src/configs.rs

```diff
@@ -221,30 +221,6 @@ pub enum WorkingMode {
     Copy,
 }
 
-impl WorkingMode {
```

Comment:
I think it will be used somewhere, but not actually used now. I'd add back this if needed